src/openrct2/scripting/IconNames.hpp

```diff
@@ -71,6 +71,16 @@ namespace OpenRCT2::Scripting
         { "rides_water", SPR_TAB_RIDES_WATER_0 },
         { "rides_thrill", SPR_TAB_RIDES_THRILL_0 },
         { "pause", SPR_TOOLBAR_PAUSE },
+        { "land_tool_decrease", SPR_LAND_TOOL_DECREASE },
```

Comment:
now I'm thinking maybe the `TransparentIconName` options should sit in another mapping to prevent unexpected values passed as `TransparentIconName`

Reply:
oh I see I accepted a number for the image back in openrct2.d.ts so I wonder how tight input validation should happen for the `image` field.